include/ur_api.h

```diff
@@ -2142,57 +2142,55 @@ typedef enum ur_device_info_t {
   /// [::ur_memory_scope_capability_flags_t] return a bit-field of atomic
   /// memory fence scope capabilities
   UR_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES = 103,
-  /// [::ur_bool_t] support for bfloat16
-  UR_DEVICE_INFO_BFLOAT16 = 104,
   /// [uint32_t] Returns 1 if the device doesn't have a notion of a
   /// queue index. Otherwise, returns the number of queue indices that are
   /// available for this device.
-  UR_DEVICE_INFO_MAX_COMPUTE_QUEUE_INDICES = 105,
+  UR_DEVICE_INFO_MAX_COMPUTE_QUEUE_INDICES = 104,
```

Comment:
hm I didn't consider that this would result in all the subsequent enum values changing, maybe we need a way to tag an enum as deprecated rather than removing it